types/office-js-preview/index.d.ts

```diff
@@ -28906,115 +28906,6 @@ declare namespace Excel {
          */
         undo = "undo"
     }
-    /**
```

Comment:
Why are all of these APIs related to document task change being deleted from the preview indext.d.ts? Have they been cancelled?

Reply:
The DocumentTask APIs are rearranged frequently by internal partners that rely on the APIs. The functionality that appears removed here should still be available, see `DocumentTask` and `DocumentTaskChange`.